src/services/newMessagesService.ts

```diff
@@ -235,6 +237,32 @@ export default class NewMessagesService {
   // THIRD_PARTY MESSAGE
   // ------------------------------
 
+  /**
+   * Retrieves the precondition of a specific Third-Party message.
+   */
+  public readonly getThirdPartyMessagePrecondition = async (
+    fiscalCode: FiscalCode,
+    messageId: NonEmptyString
+  ): Promise<
+    | IResponseErrorInternal
+    | IResponseErrorValidation
+    | IResponseErrorForbiddenNotAuthorized
+    | IResponseErrorNotFound
+    | IResponseErrorTooManyRequests
+    | IResponseSuccessNoContent
+    | IResponseSuccessJson<ThirdPartyMessagePrecondition>
+  > =>
+    pipe(
+      this.getThirdPartyMessageFnApp(fiscalCode, messageId),
+      TE.chain(message =>
+        pipe(
+          this.getThirdPartyMessagePreconditionFromThirdPartyService(message)
+        )
```

Comment:
we can remove this pipe, i think something like:

```ts
TE.chain(this.getThirdPartyMessagePreconditionFromThirdPartyService)
```
would be cleaner

Reply:
when you call a method it's better to use the arrow functions style.